examples/tests/sdk_tests/function/timeout.w

```diff
@@ -1,10 +1,14 @@
+
+/*\
+skip: true
```

Comment:
If it's skipped then why is the snapshot being created?
tools/hangar/__snapshots__/test_corpus/timeout.w_compile_tf-aws.md